### DISCUSSION TITLE
Extend thermostat card to water_heater domain

### DISCUSSION THREAD

**Discussion by cociweb** [home-assistant/architecture#1005] [General]
Hello,
Sorry, if my question is in wrong place, but it's not explicitly a feature request, not a bug, and even not an issue:  It's rather a gap, so rather all of them in the same time. But If I'm wrong, please move my ticket to the right place.

Thermostat card can handle climate domain. That is a fact. 
(MQTT) water_heater is a value added feature in 2023. Currently there is no possibility to control them from GUI (nor from external cards)

In the entity dialog box there is no issue, everything is properly visible for the water heaters but not from a lovelace thermostat card.

Is it a way/possibility to extend the inbuilt thermostat card for water_heater domain?

What is your opinion?

**Comment by MartinHjelmare** [comment]
That would probably be a water heater card.

When opening a discussion in the architecture discussions the author should have plans to implement the change discussed. As that doesn't seem to be the case here, I'll close here now.

**Reply by cociweb** [reply]
Thank you for your open discussion... coding is the next level after a decision is bornt, isn't it? 

Btw, is it really needed to duplicate a thermostat card? it address duplicated maintenance.
I still recommend to extend the thermostat card with an additional domain instead of duplicate the whole component.

**Reply by MartinHjelmare** [reply]
Do you have plans to implement this change?

**Reply by cociweb** [reply]
what change? what is the outcome? brand new card or domain extension?

**Reply by cociweb** [reply]
or any other proposal from others?

**Reply by frenck** [reply]
This is not a feature request forum. This is a place for implementors to technically propose a fundamental change in our architecture they seek approval for.

This means, that we assume you are the implementer, however, that doesn't show in the created proposal. If we misjudged that, we are sorry and in that case we like to ask you to update the proposal to contain the technical details you'd like to implement and are seeking permission for.

Otherwise, please use our community forums to request new features.

Thanks! 👍 

../Frenck
